Frontend/src/components/Connections.jsx

```
@@ -0,0 +1,87 @@
+  return (
+    <div className="max-w-2xl mx-auto bg-white shadow-lg rounded-lg p-6 mt-10">
+      <h2 className="text-2xl font-bold text-gray-800 mb-4 text-center">Your Connections</h2>
+      <ul className="space-y-2">
+        {connections.map(conn => (
+          <li key={conn.id} className="flex justify-between items-center bg-gray-100 p-3 rounded-lg shadow">
+            <span className="text-gray-800 font-medium">{conn.username}</span>
+            <button 
+              onClick={() => handleDeleteConnection(conn.id)}
+              className="bg-red-500 hover:bg-red-700 text-white font-bold py-1 px-3 rounded"
+            >
+              Remove
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)